packages/cubejs-materialize-driver/README.md

```diff
@@ -11,7 +11,7 @@ Pure Javascript Materialize driver. Based on pg driver for PostgreSQL.
 
 ## Support
 
-This driver has been contributed by [Joaquin Colacci](https://github.com/joacoc). This package is **community supported** and should be used at your own risk. 
+This driver is maintained by [Bobby Iliev](https://github.com/bobbyiliev). This package is **community supported** and should be used at your own risk.
```

Comment:
@bobbyiliev It's surely OK to add your name here, if you indeed want to maintain the driver—but why remove Joaquin's?

Reply:
Good point, the initial idea behind this was that Joaquin is no longer with the company, but now that you mention it, this does not make sense to remove his name. Just updated this.

Reply:
We should replace this with "maintained by Materialize", and point to our support e-mail and/or community Slack.